controller/handler/badge.go

```diff
@@ -216,17 +216,14 @@ func getBadgeColorFromResponseTime(responseTime int, key string, config *config.
 
 	group, name, _ := store.Get().GetGroupAndNameByKey(key)
 	ep := config.GetEndpointByGroupAndName(group, name)
+	colors := []string{badgeColorHexAwesome, badgeColorHexGreat, badgeColorHexGood, badgeColorHexPassable, badgeColorHexBad}
```

Comment:
This should probably be moved to a variable outside of this function, as it will be re-instantiated many times with the exact same values.